utkdigitalinitiatives/TRACE
# Trac 1068

** JIRA Ticket**: (TRAC-1059, TRAC-1116)
Google Scholar Inclusion requires that the name of the Author appear alone on the first line below the title on the Object Level Page for the object.

Our default first line had the (YEAR) in parentheses followed by the Title on this line.  

# What does this Pull Request do?
This pull request causes display of Author name on the first line below the title.

The file changed is $DRUPAL_HOME/sites/all/themes/UTKdrupal/template.php.

Technical details:  The Author name was already displayed on this page in the Details section of the page.
Details are on the path: 
$variables['page']['content']['system_main']['citation.tab']['metadata']['#markup']
This is a single string of HTML markup, thousands of characters.
I created a new variable $thisDetails to be able to parse and chop at this string.
Extract the Author name from $thisDetails.
Set the Author name as the value of the path:
$variables['page']['content']['system_main']['citation.tab']['citation']['#markup']
This replaces the default display.


# How should this be tested?

1.  Go to vagrant TRACE home page.
2.  Scroll down to the search by date across all collections link.
3.  On the iby page (Search by Date), click on 2008 because the default ETDS in vagrant give results for this year.
4.  After you get the search results for the 2008 search, click on a Title that will bring up an Object Level result page.
5.  On the Object Level page, you should see the Author name on the first line below the title.
6.  Please check at least 3 titles for correct display on Object Level page.


#Note 
Deleted test files from branch TRAC-1068.

# Interested parties

@DonRichards @CanOfBees
